compare.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Compares baserom.gbc and pokepinball.gbc
+
+# create baserom.txt if necessary
+
+if [ ! -f baserom.txt ]; then
+	hexdump -C baserom.gbc > baserom.txt
+fi
+
+hexdump -C pokepinball.gbc > pokepinball.txt
+
+diff -u baserom.txt pokepinball.txt | less
```

Comment:
It was extremely useful while disassembling Crystal; whenever something didn't match, this script would point me to where the mismatch is.